### ISSUE TITLE
Set5 test set evaluation result

### ISSUE THREAD

**Issue by LiUzHiAn** [hengchuan/RDN-TensorFlow#16]
Hi,
I used the model you pretrained with 200 epochs and evaluated the model on Set5 test set. But I only got ~25.5 average PSNR which is somewhat poor. Anybody same with me? Should I train the model continuely ?

**Comment by 13135675192** [comment]
hello！How long have you been training this network?

**Comment by LiUzHiAn** [comment]
I used Set91 and BSD200 as training set. Traning data 200 epochs on Colab nearly cost 5 hrs

**Comment by 13135675192** [comment]
I used DIV2K as training set. and set "matlab -true "on GTX-1080ti  nearly cost 8days.
The result is 34.5average PSNR,the testing set is set5

**Comment by LiUzHiAn** [comment]
oh, the result is not that good even worse than SRCNN. Have you got any improvement ?

**Comment by sxd0071** [comment]
@13135675192 Hey, how many training images do you have. After preprocessing the traing images, I got about 500 thousand images. When I train this network with scale of 3 for seven days , the trainning step is just 800 thousand.

**Comment by kong1060553768** [comment]
@ 13135675192 May I ask you 34.5 average PSNR at scale3 or 2?

Can you use your checkpoint folder for reference? My mailbox is 1060553768@qq.com
